README.md

```diff
@@ -63,24 +63,30 @@ Log in to GitHub and [register a new personal access token](https://github.com/s
 
 #### Next, Create a file named .env, copy the contents of the .env.template file into it, and add your values to the new file.
 - Paste the access token into GITHUB_TOKEN
-- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Typically, this would either be "Etc/UTC" or the time zone of your organization's main office, such as "America/Los_Angeles". 
+- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). TIt would normally be "local" or the time zone of the organization's main office, such as "America / Los_Angeles", or even "utc" for the system's local zone. 
```

Comment:
"TIt" is a typo (should be "It") but also, after reading this sentence, if I were a new user without much timezone experience, I wouldn't know what to put in this field. What's the default?

Also, .env.template should be changed to use that default. Currently, it still says "Etc/UTC" but I think from reading this paragraph that maybe we want the default to be "local"?

Reply:
I fixed the typo. And I clarified the section, and specifically listed the legal values. And I created a new section. And I removed the image, because its big heading was more confusing than helpful. And I corrected the example output. 

And I updated .env.template.